README.md

```diff
@@ -1,73 +1,107 @@
-# Rooch
-
-[Rooch](https://rooch.network) is a modular DApp container with the [Move language](https://github.com/move-language/move).
-
+<div align="center">
+  <br />
+  <br />
+  <a href="https://rooch.network"><img alt="Rooch" src="https://rooch.network/logo/rooch_black_combine.svg" width=384></a>
+  <br />
+  <h3><a href="https://rooch.network">Rooch</a> is Modular DApp Container, with <a href="https://github.com/move-language/move)">Move.</a></h3>
+  <br />
+</div>
 
 [![Check-Build-Test](https://github.com/rooch-network/rooch/actions/workflows/check_build_test.yml/badge.svg)](https://github.com/rooch-network/rooch/actions/workflows/check_build_test.yml)
 [![License](https://img.shields.io/badge/license-Apache-green.svg)](LICENSE)
 [![LoC](https://tokei.rs/b1/github/rooch-network/rooch?category=lines)](https://github.com/rooch-network/rooch)
 
 ## Usage
 
-1. Rooch Ethereum Layer2: Rooch(Execution) + Layer1s(Settlement) + Ethereum(Arbitration) + DA
-2. XChain Modular DApp: Rooch(Execution) + XChain(Settlement + Arbitration) + DA
-3. Rooch Layer3 Modular DApp: Rooch(Execution) + Rooch Layer2(Settlement + Arbitration) + DA
-4. Sovereign Rollup: Rooch + DA
+* **Multi-Chain Layer2:** Rooch(Execution) + Multi-Chain(Settlement) + Ethereum(Arbitration) + DA
+* **X-Chain Modular DApp:** Rooch(Execution) + X-Chain(Settlement + Arbitration) + DA
+* **Layer3 Modular DApp:** Rooch(Execution) + Rooch Layer2(Settlement + Arbitration) + DA
+* **Sovereign Rollup:** Rooch + DA
```

Comment:
The colon is placed outside the bold

Reply:
> The colon is placed outside the bold

Gotcha. You're implying that the colon need not be emboldened, correct? @geometryolife

Reply:
yeah.